Xamarin.Forms.Build.Tasks/CompiledConverters/TypeTypeConverter.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Xml;
+
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+using Xamarin.Forms.Build.Tasks;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Core.XamlC
+{
+	class TypeTypeConverter : ICompiledTypeConverter
+	{
+		public IEnumerable<Instruction> ConvertFromString(string value, ModuleDefinition module, BaseNode node)
+		{
+			if (string.IsNullOrEmpty(value))
+				goto error;
+
+			var split = value.Split(':');
+			if (split.Length > 2)
+				goto error;
+
+			XmlType xmlType;
+			if (split.Length == 2)
+				xmlType = new XmlType(node.NamespaceResolver.LookupNamespace(split[0]), split[1], null);
+			else
+				xmlType = new XmlType(node.NamespaceResolver.LookupNamespace(""), split[0], null);
+
+			var typeRef = xmlType.GetTypeReference(module, (IXmlLineInfo)node);
+			if (typeRef == null)
+				goto error;
+
+			var getTypeFromHandle = module.Import(typeof(Type).GetMethod("GetTypeFromHandle", new[] { typeof(RuntimeTypeHandle) }));
+			yield return Instruction.Create(OpCodes.Ldtoken, module.Import(typeRef));
+			yield return Instruction.Create(OpCodes.Call, module.Import(getTypeFromHandle));
+			yield break;
```

Comment:
for references, and understanding what's going on, this c# code
```
public static void Main()
{
    var a = typeof(Tests);
}
```

is compiled to
```
.method public static hidebysig 
           default void Main ()  cil managed 
    {
        // Method begins at RVA 0x2058
	.entrypoint
	// Code size 12 (0xc)
	.maxstack 1
	.locals init (
		class [mscorlib]System.Type	V_0)
	IL_0000:  ldtoken Tests
	IL_0005:  call class [mscorlib]System.Type class [mscorlib]System.Type::GetTypeFromHandle(valuetype [mscorlib]System.RuntimeTypeHandle)
	IL_000a:  stloc.0 
	IL_000b:  ret 
    } // end of method Tests::Main
```

the 2 first instructions (the 10 first bytes) is what we reproduce here